OpenSim/Common/Component.cpp

```diff
@@ -134,11 +152,19 @@ void Component::finalizeFromProperties()
 }
 
 // Base class implementation of virtual method.
-// Call extendFinalizeFromProperties on all components
+// Call finalizeFromProperties on all subcomponents
 void Component::componentsFinalizeFromProperties() const
```

Comment:
@aseth1 True that we get bad messages now, and that's why we're doing this. Already an improvement. GeometryPath is special in that it's unnamed single object, what I ran into with Geometry is that you can have multiple unnamed Mesh(es) or Sphere(s) so the class name doesn't do the trick (don't know if possible else where). ModelComponent::addGeometry generates a unique new name (was done pre naming convention).
We can open a separate issue for naming and move past this particular issue.